### PR TITLE
Package release before publishing

### PR DESCRIPTION
**Pull request by malcolmholmes**
Separates package/publish phases, for now, diverts publish destinations
to intermediate locations.

### CODE REVIEW

File: scripts/build/store-npm-packages.sh [hunk 1]
```diff
@@ -0,0 +1,19 @@
...
+fi
+
+echo "Storing NPM artifacts"
+ZIPFILE=grafana-npm-${_grafana_version}.tgz
+tar -czf "$ZIPFILE" packages/*/dist packages/*/compiled
+
+echo "${GCP_KEY}" | base64 -d > credentials.json
+gcloud auth activate-service-account --key-file=credentials.json
+gsutil cp "$ZIPFILE" "gs://${PRERELEASE_BUCKET}/artifacts/npm/$ZIPFILE"
+echo "Done."
```
Comment (dsotirakis) [inline scripts/build/store-npm-packages.sh:19]: Can we handle an error here and be more descriptive, like `Successfully stored NPM artifacts` etc?

File: scripts/drone/pipelines/release.star [hunk 2]
```diff
@@ -226,7 +279,7 @@ def release_pipelines(ver_mode='release', trigger=None):
             steps=[download_grabpl_step()] + initialize_step(edition='oss', platform='linux', ver_mode=ver_mode, install_deps=False) + steps,
             depends_on=[p['name'] for p in oss_pipelines + enterprise_pipelines],
         )
-        pipelines.append(publish_pipeline)
+        #pipelines.append(publish_pipeline)
```
Comment (dsotirakis) [inline scripts/drone/pipelines/release.star:282]: This can be now removed?

**Review by dsotirakis — APPROVED** [review]
🚀  🚀

File: scripts/build/store-npm-packages.sh (hunk 1 above)
Comment (malcolmholmes) [reply scripts/build/store-npm-packages.sh]: `set -e` handles errors, tells script to fail on errors. Have improved the message.

File: scripts/drone/pipelines/release.star (hunk 2 above)
Comment (malcolmholmes) [reply scripts/drone/pipelines/release.star]: yep

**Pull request merged**